typescript/web/src/connectors/resolvers/__tests__/export-to-coco.ts

```diff
@@ -133,7 +133,7 @@ const createLabelWithLabelClass = (imageId: string, labelClassId: string) => {
   });
 };
 
-describe("Exporting a dataset to coco format", () => {
+describe.skip("Exporting a dataset to coco format", () => {
```

Comment:
It would be nice to reactivate those tests

Reply:
Finally I removed those tests, because the output of the resolvers is now a URL so it is more intricate to get the content of the file from that, and given that it's even more difficult when dealing with yolo format since it's a zip with several files. Core functions are tested in `typescript/common-resolvers/src/export/format-yolo/__tests__/export-to-yolo.ts` and `typescript/common-resolvers/src/export/format-coco/coco-core/__tests__/converters.ts`.